Rubberduck.Resources/RubberduckUI.de.resx

```diff
@@ -990,7 +990,12 @@ Möchten sie die Einstellungen in Rubberduck importieren?</value>
     <value>Fenstersichtbarkeit beim Start</value>
   </data>
   <data name="VersionCheck_NewVersionAvailable" xml:space="preserve">
-    <value>Rubberduck Version {0} ist jetzt verfügbar! Wollen sie jetzt die Release Notes prüfen?</value>
+    <value>Sie verwenden momentan Rubberduck Version {0}.
+Version {1}({2}) ist nun verfügbar.
+
+Um den Change Log anzuschauen und Rubberduck manuell abzudaten, klicke 'Ja'.
```

Comment:
"[..] <s>abzudaten</s> zu aktualisieren, klicke**n Sie** 'Ja'."

Reply:
Danke, _aktualisieren_ wollte mir einfach nicht in den Sinn kommen.